services/api-server/src/simcore_service_api_server/services/director_v2.py

```diff
@@ -116,8 +84,9 @@ async def create_computation(
             },
         )
         response.raise_for_status()
-        return ComputationTaskGet(**response.json())
+        return ComputationTaskGet.parse_raw(response.text)
```

Comment:
actually I am also wondering a bit if we need parsing/validation here at all.. using the .construct could improve speed. but of course we would need to see if we have loop freezing at all or not. @pcrespov?

Reply:
Since we have now shared models between services (i.e. `models_library.api_schema_...` family) the risk is much smaller (only if different version of services are in place might cause trouble but unlikely to happen except perhaps in a redeploy) so yes, one could assume that.

Reply:
I will leave this as is according to the discussion we had above.